### PR TITLE
[PRM-411]: added agent messages for reasonable excuse journey

### PR DESCRIPTION
**Pull request by BackRowOla**
Honesty declaration agent text changes for:
- Crime
Screenshot 2021-07-19 at 12 35 28

- Bereavement
Screenshot 2021-07-19 at 12 36 24

Url text changes on confirmation page
Screenshot 2021-07-19 at 12 37 16

### CODE REVIEW

File: app/utils/MessageRenderer.scala [hunk 1]
```diff
@@ -21,8 +21,15 @@ import play.api.i18n.Messages
 
 object MessageRenderer {
   def getMessage(msgKey: String, msgArgs: Any*)(implicit messages: Messages, user: UserRequest[_]): String = {
+
     if (user.isAgent){
-      messages.apply(s"agent.$msgKey", msgArgs: _*)
+      if(user.session.get(SessionKeys.whoPlannedToSubmitVATReturn).contains("client") ||
+        (user.session.get(SessionKeys.whoPlannedToSubmitVATReturn).contains("agent") && user.session.get(SessionKeys.causeOfLateSubmissionAgent).contains("client"))){
```
Comment (ruthdas03) [inline app/utils/MessageRenderer.scala:27]: So can we not store the 'user.session.get(SessionKeys.whoPlannedToSubmitVATReturn)' in a String val and do the 'contains' as 'client' or 'agent'.

Comment (BackRowOla) [reply app/utils/MessageRenderer.scala:27]: Made that change locally yesterday, makes it easier to read 🙂

File: conf/messages [hunk 2]
```diff
@@ -79,11 +79,15 @@ honestyDeclaration.heading = Honesty declaration
 honestyDeclaration.p1 = I confirm that:
 honestyDeclaration.li.1.1 = because {0},
 honestyDeclaration.li.1.2 = I was unable to submit the VAT Return due on {0}
+agent.honestyDeclaration.li.1.2 = they were unable to submit the VAT Return due on {0}
 honestyDeclaration.li.2 = no one else was available to make the submission for me
+agent.honestyDeclaration.li.2 = no one else was available to make the submission for them
```
Comment (ruthdas03) [inline conf/messages:84]: How are testing this message (or similar ones) like we have tested for messages in case of agent and VAT trader for 'crime'.

Comment (ruthdas03) [reply conf/messages:84]: Please ignore, I think it's done in 'li2AgentText'

**Review by ruthdas03 — COMMENTED** [review]
Looks ok to me excepting that we are not considering the error message for agent and VAT Trader separately.

**Review by LewisMitchell — APPROVED** [review]
LGTM. Waiting on AT change

**Pull request merged**